extensions/sql-migration/src/api/sqlUtils.ts

```diff
@@ -153,6 +179,7 @@ function getSqlDbConnectionProfile(
 			password: password,
 			connectionTimeout: 60,
 			columnEncryptionSetting: 'Enabled',
+			// when connecting to a target Azure SQL DB, use true/false
```

Comment:
collecting target database table info for SQL DB scenario - verify that true/true is correct for the target

Reply:
cc @siyangMicrosoft

Reply:
we should only connect to azure targets with encrypt and no trust

Reply:
same for other connections below...

Reply:
looks like true/false was correct for DB and MI, resolving this thread, but will address true/false not working for VM elsewhere